### ISSUE TITLE
Crash when syncing folder on Android

### ISSUE THREAD

**Issue by MaxG87** [nextcloud/android#6093]
### Steps to reproduce
1. Create and populate folder on second device, e.g. PC
2. Switch to Android device with NextCloud Dev.
3. Try to synchronise whole folder.

### Expected behaviour
Folder should sync.

### Environment data
Device model: 
Pocophone F1

#### Nextcloud log (data/nextcloud.log)
```
************ CAUSE OF ERROR ************

java.lang.RuntimeException: Unable to start service com.owncloud.android.services.OperationsService@571ea0b with Intent { act=SYNC_FOLDER cmp=com.nextcloud.android.beta/com.owncloud.android.services.OperationsService (has extras) }: java.lang.ClassCastException: com.nextcloud.client.account.RegisteredUser cannot be cast to android.accounts.Account
	at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:4142)
	at android.app.ActivityThread.access$1900(ActivityThread.java:220)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1915)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loop(Looper.java:224)
	at android.app.ActivityThread.main(ActivityThread.java:7520)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:539)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:950)
Caused by: java.lang.ClassCastException: com.nextcloud.client.account.RegisteredUser cannot be cast to android.accounts.Account
	at com.owncloud.android.services.OperationsService.onStartCommand(OperationsService.java:196)
	at android.app.ActivityThread.handleServiceArgs(ActivityThread.java:4121)
	... 8 more

************ APP INFORMATION ************
ID: com.nextcloud.android.beta
Version: 20200423
Build flavor: versionDev

************ DEVICE INFORMATION ************
Brand: Xiaomi
Device: beryllium
Model: POCOPHONE F1
Id: QKQ1.190828.002
Product: beryllium

************ FIRMWARE ************
SDK: 29
Release: 10
Incremental: V11.0.4.0.QEJMIXM```

**Comment by ezaquarii** [comment]
@tobiasKaminsky I believe this has been fixed in dev?

**Comment by AndyScherzinger** [comment]
I believe so too, please update your client, version April 24th is very old.

**Comment by AndyScherzinger** [comment]
Already fixed here: #6046
